webapp/tests/client_websocket.test.jsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+/*
```

Comment:
Commented out because? I thought this was supposed to fix the client's ability to test the websocket.